server/tasks/register/test.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (grunt) {
+	grunt.registerTask('test', [
+		'mochaTest'
+	]);
+};
```

Comment:
Good catch. That's what I get for copying from an example.  Thought I had sublime set up to to replace them. Thanks again.